Gruntfile.js

```diff
@@ -0,0 +1,303 @@
+/* jshint node:true */
+const glob = require('glob');
+const sass = require('node-sass');
+
+// variables
+const sourceFolderRelPath = 'src';
+const destFolderRelPath = 'assets';
+
+// add js files here if necessary
+const sourceJsFiles = [
+	`${sourceFolderRelPath}/**/*.js`,
+];
+
+// methods
+const getFilePaths = (pathArr) => {
+	let filePaths = [];
+	for (let path of pathArr) {
+		const files =  glob.sync(path);
+		filePaths = filePaths.concat(files);
+	}
+	return filePaths;
+};
+
+const getFilesPathObjects = (pathArr) => {
+	const filesPathsObjs = [];
+	for (let path of pathArr) {
+		const src = path;
+		const dest = path.replace(sourceFolderRelPath, destFolderRelPath);
+		filesPathsObjs.push({src, dest});
+	}
+	return filesPathsObjs;
+};
+
+const jsFilesPaths = getFilePaths(sourceJsFiles);
+const jsFilesPathsObjs = getFilesPathObjects(jsFilesPaths);
+
+/**
+ * Returns an object with destinations as key and sources as value
+ * @param filesPathsObjs
+ * @returns {}
+ */
+const buildFileConfiguration = (filesPathsObjs) => {
+	let fileConfig = {};
+	for (let pathObj of filesPathsObjs) {
+		fileConfig[pathObj.dest] = pathObj.src;
+	}
+	return fileConfig;
+};
+
+module.exports = function( grunt ) {
+	'use strict';
+
+	grunt.initConfig({
+
+		// JavaScript linting with JSHint.
+		jshint: {
+			options: {
+				jshintrc: '.jshintrc'
+			},
+			all: jsFilesPaths.concat('Gruntfile.js')
+		},
+
+		browserify: {
+			dist: {
+				files: buildFileConfiguration(jsFilesPathsObjs)
+			},
+			options: {
+				transform: [['babelify', {
+					presets: [
+						'@wordpress/babel-preset-default',
+					]
+				}]],
+				browserifyOptions: {
+					debug: true
+				}
+			}
+		},
+
+		// Minify .js files.
+		uglify: {
+			options: {
+				preserveComments: 'some',
+				sourceMap: true
+			},
+			main: {
+				files: [{
+					expand: true,
+					src: jsFilesPathsObjs.map(file => file.dest),
+					dest: './',
+					ext: '.min.js',
+					extDot: 'last',
+				}]
+			}
+		},
+
+		// Compile all .scss files.
+		sass: {
+
+			dist: {
+				options: {
+					require: 'susy',
+					implementation: sass,
+					sourceMap: true,
+					includePaths: ['node_modules/susy/sass'].concat( require( 'node-bourbon' ).includePaths )
+				},
+				files: [
+					{
+						expand: true,
+						cwd: `${sourceFolderRelPath}/scss`,
+						src: ['**/*.scss¿'],
```

Comment:
Thanks @nielslange !

I noticed the CSS map files weren't generated and then can see an unusual ¿ character here, wondering if that is the cause?

Reply:
> I noticed the CSS map files weren't generated and then can see an unusual ¿ character here, wondering if that is the cause?

Yes, that causes the problem. The ¿ character should not be there. I'll update this PR accordingly and will also adjust the plugin header as requested in https://github.com/woocommerce/storefront-footer-bar/pull/12#discussion_r495521920.

Reply:
Fixed 😀